#request constants
string loc
bool is_pick
---
#response constants
int8 status